sdk/identity/identity/review/identity.api.md

```diff
@@ -138,6 +138,13 @@ export class UsernamePasswordCredential implements TokenCredential {
     getToken(scopes: string | string[], options?: GetTokenOptions): Promise<AccessToken | null>;
     }
 
+// @public (undocumented)
+export class VSCodeCredential implements TokenCredential {
```

Comment:
The plan for this as of now is to not export this type, but to include it into the DefaultAzureCredential as you have.